Test/Stats/test_stats.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import pytest
+
+import netket as nk
+import netket.variational as vmc
+from netket.operator import local_values
+from netket.stats import statistics
+
+
+def _setup():
+    g = nk.graph.Hypercube(3, 2)
+    hi = nk.hilbert.Spin(g, 0.5)
+
+    ham = nk.operator.Heisenberg(hi)
+
+    ma = nk.machine.RbmSpinV2(hi, alpha=2)
+    ma.init_random_parameters()
+
+    return hi, ham, ma
+
+
+def _test_stats_mean_std(hi, ham, ma, batch_size):
+    sampler = nk.sampler.MetropolisLocal(ma, batch_size=batch_size)
+
+    n_samples = 16000
+    num_samples_per_chain = n_samples // batch_size
+
+    res = vmc.compute_samples(sampler, n_samples=n_samples, n_discard=6400)
+    assert res.samples.shape == (num_samples_per_chain, batch_size, hi.size)
+
+    eloc = local_values(res.samples, res.log_values, ma, ham)
+    assert eloc.shape == (num_samples_per_chain, batch_size)
+
+    stats = statistics(eloc)
+
+    # These tests only work for one MPI process
+    assert nk.MPI.size() == 1
+
+    assert stats.mean == pytest.approx(np.mean(eloc))
+    if batch_size > 1:
+        # error of mean == stdev of sample mean between chains / sqrt(#chains)
```

Comment:
Did you check this after I fixed the issue with the factor in error of mean in C++?

Reply:
Yes, I changed `ddof=0` (it was 1 previously because of the `m-1` denominator) and added the division by `sqrt(m)`. This matches the error of mean computed with your changes.